tendrl/commons/objects/__init__.py

```diff
@@ -167,8 +168,7 @@ def load_all(self):
                         NS._int.reconnect()
                         etcd_resp = NS._int.client.read(value)
                     else:
-                        return None
```

Comment:
Please use etcd_utils.read instead of NS._int.client.read, also you dont need to handle etcd.EtcdConnectionFailed, etcd.EtcdException, just  "etcd.EtcdKeyNotFound" should be handled